.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "examples/NonArduino/Tock/libtock-c"]
```

Comment:
This submodule will not be available if someone installs RadioLib as an Arduino Library (via Arduino library manager). Probably not an issue, but I wanted to mention it.